style.css

```diff
@@ -0,0 +1,203 @@
+body {
+  box-sizing: border-box;
+  background-color: #979797;
+  padding: 2.4rem;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 62.5%;
+  font-weight: 700;
+}
+
+.header {
+  width: 80rem;
+  margin: 0 auto;
+}
+
+.wrapper {
+  margin: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+h1 {
+  display: flex;
+  justify-content: center;
+}
+
+.book-container {
+  display: block;
+  padding: 1rem;
+  border: 3px solid #1a1a1a;
+  width: auto;
+  margin-left: 20%;
+  margin-right: 20%;
+  background-color: white;
+}
+
+.books-list {
+  box-sizing: border-box;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  align-items: center;
+  width: 100%;
+}
+
+.form-control {
+  height: 40px;
+}
+
```

Comment:
- [ ] [OPTIONAL] Great job with your project 👍 . Please increase the font size of your `input` and the text on the list page as it can be really hard for a user with eye problems to see. 🙏 

Add New Page | List page
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177961796-8f8e62ac-d34c-4b26-b235-c10f56900119.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177961837-97deee56-1f35-4302-9b09-ec5a3546d1de.png)